charts/galoy/values.yaml

```diff
@@ -97,24 +97,22 @@ redis:
   metrics:
     enabled: true
 admin-panel:
-  enabled: false
+  enabled: true
+  nameOverride:
+  image:
+    repository: us.gcr.io/galoy-org/admin-panel
+    digest: "sha256:bd4f821f3c7e2cff57361f2f52ef85855585c42c997750a37ef7df970cfb3377"
+    git_ref: "3666aa3" # Not used by helm
+  labels: {}
   service:
     type: ClusterIP
-  cloneStaticSiteFromGit:
-    enabled: true
-    repository: https://github.com/GaloyMoney/admin-panel.git
-    branch: build
-  podSecurityContext:
-    enabled: true
-    runAsUser: 1001
-    # there seems to be inconsistency on sysctls on bitnami
-    # sometimes it's a map, sometimes an array
-    sysctls: []
+    port: 80
   ingress:
     enabled: false
-  podLabels:
-    app: admin-panel
-  command: ['/bin/sh', '-c', 'echo "window.env={\"GRAPHQL_URI\":\"https://graphql.{{.Values.global.network}}.galoy.io/graphql\"}" > /app/env.js && /opt/bitnami/scripts/nginx/entrypoint.sh /opt/bitnami/scripts/nginx/run.sh']
+    certManager: false
```

Comment:
This doesn't make sense to me - why we setting certManager = false but adding the annotations? I think the boundaries are wrong. The annotations should be in the ingress definition or?